packages/lesswrong/server/scripts/triggerWrappedRefresh.ts

```diff
@@ -11,7 +11,7 @@ import { Globals } from "../vulcan-lib";
 //   state
 // FROM pg_stat_activity order by duration desc;
 
-const USER_ENGAGEMENT_VIEW_NAME = 'user_engagement_wrapped_2023';
+const USER_ENGAGEMENT_VIEW_NAME = 'user_engagement_wrapped_2022';
```

Comment:
This has a year in the name, but it's actually all data **starting** from that year, and we query it with a specific year filter

Reply:
I think just `user_engagement_wrapped` would be clearer. I guess changing it would break the /wrapped2022 route until this is deployed, but that seems ok